### PR TITLE
docs: changes to the docs as discussed in #3460

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
Changes to the docs as discussed with Giovanni and mentioned in this issue:
https://github.com/aquasecurity/trivy/issues/3460

### CODE REVIEW

**Comment by AnaisUrlichs** [comment]
@itaysk @knqyf263 when you have a sec to review this :)

**Comment by AnaisUrlichs** [comment]
@knqyf263 @giorod3 could you please review?

File: pkg/fanal/analyzer/analyzer.go [hunk 1]
```diff
@@ -454,6 +454,9 @@ func (ag AnalyzerGroup) AnalyzeFile(ctx context.Context, wg *sync.WaitGroup, lim
 }
 
 func (ag AnalyzerGroup) RequiredPostAnalyzers(filePath string, info os.FileInfo) []Type {
+	if info.IsDir() {
```
Comment (knqyf263) [inline pkg/fanal/analyzer/analyzer.go:457]: You may have failed to merge the main branch.

File: docs/getting-started/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Quick Start
```
Comment (knqyf263) [inline docs/getting-started/quick-start.md:1]: Looks like duplicated
https://github.com/aquasecurity/trivy/blob/a67955a0040a47eb363d8f5c9a8c44096215dd0c/docs/index.md#get-trivy

Comment (AnaisUrlichs) [reply docs/getting-started/quick-start.md:1]: I am not sure why Dimitry's work is getting merged in...

File: pkg/fanal/analyzer/analyzer.go (hunk 1 above)
Comment (AnaisUrlichs) [reply pkg/fanal/analyzer/analyzer.go:457]: what did I merge then... AYyy

Comment (knqyf263) [reply pkg/fanal/analyzer/analyzer.go:457]: I recommend using `git merge main`. We squash and merge commits in PR. You don't have to care about linear history.

**Pull request closed without merging**